apps/src/netsim/controls.html.ejs

```diff
@@ -1,3 +0,0 @@
-<div id="slider-cell">
-  <img id="spinner" style="visibility: hidden;" src="<%= assetUrl('media/netsim/loading.gif') %>" height=15 width=15>
-</div>
```

Comment:
Don't think so. I think I removed a bunch, but evidently missed a couple. I'll make a pass of the remaining ones in a separate PR.